### ISSUE TITLE
Solaris 11 fails for test_fx

### ISSUE THREAD

**Issue by quanah** [heimdal/heimdal#317]
```
test_fx: key not same for enctypes 18 18 18
test_fx: key not same for enctypes 18 18 17
test_fx: key not same for enctypes 17 18 18
test_fx: KRB-FX-CF2 not supported for enctype 1: kerberos prf for des-cbc-crc not supported
test_fx: key not same for enctypes 16 16 16
test_fx: 4 KRB-FX-CF2 vectors failed
FAIL: test_fx
```

**Comment by sfhacker** [comment]
Just built master branch from source and got this:

test_fx: KRB-FX-CF2 not supported for enctype 1: kerberos prf for des-cbc-crc not supported
PASS test_fx.exe (exit status: 0)

**Comment by lhoward** [comment]
Fixed in a88d0066.
